client/boot/index.js

```diff
@@ -87,7 +88,15 @@ function setUpContext( layout ) {
 		var parsed = url.parse( location.href, true );
 
 		context.layout = layout;
-		context.reduxStore = reduxStore;
+		//( function() {
+		//	var Dispatcher = require( 'dispatcher' );
+		//	Dispatcher.register( function( payload ) {
+		//		var action = payload.action;
+		//		console.log( 'Flux -> Redux' );
+		//		reduxStore.dispatch( action );
+		//	} );
+		//}() );
```

Comment:
Interesting. Did this work? :)

I'm thinking we probably don't need to integrate the two dispatchers now, as the Redux transition is getting some momentum behind it. We should check we're not relying on any 'external' actions, although from what I can remember, we're not.